espnet/nets/pytorch_backend/e2e_asr.py

```diff
@@ -392,21 +392,21 @@ def subsample_frames(self, x):
 
 
 # TODO(pzelasko): Currently allows half-streaming only; needs streaming attention decoder implementation
-class StreamingE2E(object):
+class WindowStreamingE2E(object):
```

Comment:
can you put this class into a new module? for example `espnet.nets.pytorch_backend.streaming.window`